l10n_it_fatturapa_in/views/account_view.xml

```diff
@@ -104,6 +104,12 @@
         <field name="model">account.invoice</field>
         <field name="inherit_id" ref="account.invoice_supplier_form"></field>
         <field name="arch" type="xml">
+            <xpath expr="//div[@role='alert']" position="after">
+                <div class="alert alert-info" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('e_invoice_validation_error','=',False)]}">
+                     <bold>The invoice doesn't match one of the amount untaxed, tax amount or total amount of the original e-invoice!!! Please check</bold>
+                </div>
```

Comment:
Un punto esclamativo era sufficiente secondo me, cmq ok :-)

Reply:
@eLBati Quindi rimane così?

Reply:
Come preferisci

Reply:
Direi che un punto esclamativo è più che sufficiente nella stringa sorgente in inglese.

Tra l'altro poi delle traduzioni si tende ad evitare l'uso dei punti esclamativi e interrogativi, vedi linee guida:

https://digilander.libero.it/elleuca/linee-guida/stile.xhtml

Reply:
@eLBati @primes2h Ditemi cosa scrivere così tagliamo le iterazioni ;)

Reply:
@robyf70 Mettila così, in svariati anni di traduzione software non ho mai trovato stringhe sorgenti con più di un punto esclamativo. :wink:

Reply:
@primes2h @eLBati ... a me è capitato il contrario ma comunque ogni mondo è paese ... ho pushato senza punti esclamativi :)

Reply:
@robyf70 
:open_mouth: 
Non ho capito perché siamo passati dall'indicazione di metterne uno a non metterne nessuno. :laughing: 

Forse mi sono spiegato male, nelle stringhe sorgenti in inglese "si usa" il punto esclamativo. È nelle successive traduzioni in italiano che si tende invece a non usarlo, ma è una cosa da valutare nel contesto. :-)

A questo punto fai come preferisci :-)